docs/lifecycle/lifecycle.rst

```diff
@@ -40,13 +46,16 @@ Project State Transitions
 +--------------+-------------------+----------------------+-------------------+
 | From State   | To State          | TAC Review           | Board Review      |
 +==============+===================+======================+===================+
-| Non-LFN      | Non-TAC           | LFN Entry Review     | LFN Entry Review  |
+| none         | Sandbox           | LFN Entry Review     | LFN Entry Review  |
+|              |                   | Quarterly Health Rvw |                   |
++--------------+-------------------+----------------------+-------------------+
+| Sandbox      | Incubation        | Incubation Review    | Incubation Review |
 +--------------+-------------------+----------------------+-------------------+
-| Non-TAC      | TAC               | TAC Admission Review |                   |
+| Incubation   | TAC               | TAC Admission Review | TAC Admission Rvw |
```

Comment:
is there any possibility to move from TAC to Incubation
- board priority review
- decrease of community activity (but still some => so no exit or archive)
- non respect of the criteria used for promotion (increasing overlap with other projects?)

Reply:
Seems reasonable... are these the right criteria?  Who should initiate it?

Reply:
Added an Incubation Reversal Review, modeled after the Archive Review